runtime/pkg/email/templates/scheduled_report.mjml

```diff
@@ -41,9 +47,9 @@
         <mj-spacer height="20px" />
         
         <mj-text align="center" font-size="12px">
-          To edit or unsubscribe from this report, <a href="{{ .EditLink }}">click here</a>.
+          To{{ if not .External}} edit or{{ end }} unsubscribe from this report, <a href="{{ .EditLink }}">click here</a>.
```

Comment:
Added this explicit flag other way is to check if edit link contains `token` but for that will need to define functions to convert `template.URL` to string and also add then use string.contains function. Both these function will need to be added to template func map. So thought this is much cleaner and will also perform better.